lib/openshift/cluster_logging.rb

```diff
@@ -53,7 +76,12 @@ class ClusterLogging < ClusterResource
     end
 
     private def es_cluster_health(user: nil, cached: true, quiet: false)
-      es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth']
+      @result = es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth']
+      if @result
+        es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth']
+      else
+        es_status_raw(user: user, cached: cached, quiet: quiet).first['cluster']['status']
+      end
```

Comment:
Instance variables should not be used here. A simpler way to do this would be:
```
private def es_cluster_health(user: nil, cached: true, quiet: false)
  es_status_raw(user: user, cached: cached, quiet: quiet).first['clusterHealth'] ||
    es_status_raw(user: user, cached: true, quiet: quiet).first['cluster']['status']
```

This will also avoid an extra API call (notice `cached: true` on second line.